### DISCUSSION TITLE
Emoji input panel

### DISCUSSION THREAD

**Discussion by lucasjinreal** [flyerhq/flutter_chat_ui#156] [Ideas & Requests] [answered]
if there is a emoji input panel, that would be very great. Seems, it can place a emoji panel switch button right along with attachment button

**Comment by demchenkoalex** [comment]
Are these panels are part of the iOS/Android systems? We will not be creating one from scratch that's for sure, since emoji can be selected from the keyboard, switching to the emoji one

**Comment by lucasjinreal** [comment]
@demchenkoalex  think about linux.. windows..

**Comment by lucasjinreal** [comment]
I understand we want keep this lib more simplified, but sometimes users really want a quick action with eomji even don't want call out keyboard and switching...

**Reply by demchenkoalex** [reply]
well you did a great point above, think about making this for linux, windows and web :D it's like another big project

**Reply by lucasjinreal** [reply]
I just request a tiny feature, do or not doesn't effect this lib's wonderfulness.

**Comment by demchenkoalex** [comment, marked as answer]
pointing to this https://github.com/flyerhq/flutter_chat_ui/discussions/471
